libs/cart/src/drivers/magento/queries/add-cart-item.ts

```diff
@@ -12,13 +15,15 @@ export const addSimpleCartItem = gql`
     }) {
       cart {
         ...cart
+        ${daffBuildFragmentNameSpread(...extraCartFragments)}
```

Comment:
Should this be a spread?

Reply:
yeah the util functions have rest params